src/main/scala/com/github/unchama/seasonalevents/seizonsiki/Seizonsiki.scala

```diff
@@ -0,0 +1,25 @@
+package com.github.unchama.seasonalevents.seizonsiki
+
+import java.time.LocalDate
+
+import com.github.unchama.seasonalevents.SeasonalEventsConfig
+import com.github.unchama.seasonalevents.Util.localDateFromYearMonthDays
+import com.github.unchama.seichiassist.SeichiAssist
+
+class Seizonsiki(private val plugin: SeichiAssist)(implicit config: SeasonalEventsConfig) {
+  private val today = LocalDate.now()
+  // イベント開催中か判定
+  if (today.isBefore(Seizonsiki.END_DATE)) {
+    Seizonsiki.isInEvent = true
+    plugin.getServer.getPluginManager.registerEvents(new SeizonsikiListener(), plugin)
+  }
+}
+
+object Seizonsiki {
+  var isInEvent = false
```

Comment:
```suggestion
  def isInEvent: Boolean = LocalDate.now().isBefore(Seizonsiki.END_DATE))
```
0時に終わらせるためにはこうすればよいのでは？

Reply:
...なるほど！ありがとうございますー！